SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -193,10 +193,36 @@ then
      fi
 fi
 
+boot_configuration_file="/etc/default/grub"
+header="# cmdline snippet added by %{name}-%{version}-%{release}"
+
+# Check that the configuration file exists
+if [ ! -f "$boot_configuration_file" ]; then
+    echo "Warning: $boot_configuration_file does not exist"
+    exit 0
+fi
+
+sed -i "/$header/{N;N;d;}" $boot_configuration_file
+
+
 %post
 /sbin/depmod -a %{uname_r}
 ln -sf linux-%{uname_r}.cfg /boot/mariner-mshv.cfg
 
+boot_configuration_file="/etc/default/grub"
+parameter="audit=0 console=ttyS0,115200n8 earlyprintk"
```

Comment:
might be worth discussing how we intend to handle images with multiple, varied, boot paths. 

For instance, to support kernel-mshv we will need an entirely custom menuentry that mkconfig certainly doesn't know how to generate. So, we could append them here like other packages (thus modifying the behavior of the standard boots) or hardcode the values into grub.d/40_mshv.